lib/commons/builder/config.rb

```diff
@@ -30,4 +30,12 @@ def languages
   def country_wikidata_id
     @country_wikidata_id ||= values[:country_wikidata_id]
   end
+
+  def to_liquid
```

Comment:
So, the liquid documentation isn't very clear on this, but here we are:
If you pass any object in to a liquid context, liquid expects to be able to call `to_liquid` on it. It injects this method — returning `self` — onto all the "primitive" types (Hash, Array, String, numbers, etc), so that they work as expected. `to_liquid` can't just return self, as liquid (by design) doesn't allow method calls (except for `first`, `last`, `size`).
If we don't have this method on `Config`, we would instead have to turn a config into a hash when before constructing the context to pass to `template.render!`, and pass the hash instead. Given the choice between having that logic in `WikidataQueries` or `Config`, `Config` seemed better as then adding new config parameters would be done in a single class.
In retrospect, we could rename `to_liquid` to `to_h` (or even `as_json`, to show that any nested values are also JSON primitives and thus liquid-friendly) and then use `config.to_h`/`config.as_json` in `WikidataQueries` when building the context.
I'd say it's reasonable to keep in this PR as it directly supports the aims of the PR, but I'd be happy with moving adding this method out into a separate commit.

Reply:
Great - a new commit with that nice explanation as the message would be great :+1: 

(I'm fine with `to_liquid` instead of having to explicitly call `to_h` or `as_json`.)

Reply:
:ok:!